exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -212,7 +213,25 @@ export async function handler (
           result[prefix].duration = getExecutionDuration(startTime)
         } catch (err: any) { // eslint-disable-line
           if (await isErrorCommandNotFound(params[0], err)) {
-            err.hint = buildCommandNotFoundHint(params[0], (await readProjectManifestOnly(opts.dir)).scripts)
+            err.message = `Command "${params[0]}" not found`
+            if (opts.implicitlyFellbackFromRun) {
+              try {
```

Comment:
why is this try needed? There is already a try/catch in getNearestProgram

Reply:
Because I think `readProjectManifestOnly` interacts with the filesystem and therefore can fail.